pkg/api/routes.go

```diff
@@ -60,6 +60,12 @@ func (h *Handler) GetRoutes() []Route {
 			h.CreateImage,
 		},
 		{
+			"DeleteImage",
+			"POST",
```

Comment:
The volume plugin will be calling the REST API directly right, not through the rook client? i originally thought the URI/verb wasn't a big deal since it's abstracted behind the rook client, but if you are hitting the URI directly via a separate http client and not through a rook client, then i'd say it is more important to get the URI correct right now...